p2d/tex_utilities.py

```diff
@@ -77,6 +85,7 @@ def generate_statement_tex(problem, tex_dir):
         'LEGEND': problem['statement']['legend'],
         'INPUT': problem['statement']['input'],
         'OUTPUT': problem['statement']['output'],
+        'INTERACTION': '' if problem['statement']['interaction'] is None else problem['statement']['interaction'],
```

Comment:
It is more appropriate to move this (None -> '') to parse_polygon_package because this is "morally" about parsing the package into a nice object.